app/views/pages/home/_location_categories.html.haml

```diff
@@ -1 +1,14 @@
-%h1.govuk-heading-xl Location Categories
+%h1.govuk-heading-xl #{ I18n.t('pages.home.location_links_heading') }
+.govuk-grid-row
+  .govuk-grid-column-one-half
+    %h2.govuk-heading-m #{ I18n.t('pages.home.counties') }
```

Comment:
this should be
```
t('.counties')
```
and the translation should be put in
```
pages.home.location_categories.counties
```

Reply:
resolvedaroo 👍